mono/metadata/gc.c

```diff
@@ -564,6 +564,17 @@ ves_icall_System_GC_KeepAlive (MonoObjectHandle obj, MonoError *error)
 	 */
 }
 
+void
+ves_icall_System_GC_GetGCMemoryInfo (void* klass,
+								     gint64* high_memory_load_threshold_bytes,
```

Comment:
The indentation is screwed up.

Reply:
What is the extra 'klass' argument ?

Reply:
Actually, now that I think about it more, I am not sure what that first pointer is? I am getting six pointers, but there are only five ref parameters in the C# declaration of the internal function. What is the first pointer?

Reply:
Its supposed to be the this pointer for instance icalls, but this icall is defined as static, so no idea.